e2e/index.e2e.test.js

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '../playwright/fixtures'; 
+
+test('E2E Test: Open extension popup, display greeting, and modify it on button click', async ({ context, extensionId }) => {
```

Comment:
Should we move all the tests under `/tests`

Reply:
Do you mean : tests/unit/... and tests/e2e/..? If yes, then I can reorganize and add the mocks folder as well.

Reply:
Yep! Thanks!